modules/gltf/gltf_document.cpp

```diff
@@ -6999,3 +7004,15 @@ Error GLTFDocument::append_from_file(String p_path, Ref<GLTFState> r_state, uint
 	ERR_FAIL_COND_V(err != OK, ERR_PARSE_ERROR);
 	return err;
 }
+
+Error GLTFDocument::_parse_gltf_extensions(Ref<GLTFState> state) {
+	ERR_FAIL_NULL_V(state, ERR_PARSE_ERROR);
+	if (state->json.has("extensionsRequired") && state->json["extensionsRequired"].get_type() == Variant::ARRAY) {
+		Array extensions_required = state->json["extensionsRequired"];
+		if (extensions_required.find("KHR_draco_mesh_compression") != -1) {
+			ERR_PRINT("glTF2 extension KHR_draco_mesh_compression is not supported.");
+			return ERR_PARSE_ERROR;
```

Comment:
Could maybe be `ERR_UNAVAILABLE` (i.e. unsupported)?
https://github.com/godotengine/godot/blob/cb7daddbebac8dbb354e0e92f8e16013fcb22055/core/error/error_list.h#L49

Reply:
Will amend.